pkgs/tools/X11/xplugd/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, stdenv
+, xorg
+, pkgconfig
```

aliases aren't allowed in nixpkgs
```suggestion
, pkg-config
```

---

Thank you, done